### PR TITLE
[hud][ch] Use CH queued_jobs_by_label

### PR DESCRIPTION
**Pull request by kit1980**
The query takes ~40s according to Monitoring / Query insights.

### CODE REVIEW

File: torchci/clickhouse_queries/queued_jobs_by_label/query.sql [hunk 1]
```diff
@@ -1,6 +1,4 @@
 --- This query is used by HUD metrics page to get the list of queued jobs grouped by their labels
-SET
```
Comment (malfet) [inline torchci/clickhouse_queries/queued_jobs_by_label/query.sql:2]: Q: had something like that be detected by linter of sorts?

File: torchci/clickhouse_queries/queued_jobs_by_label/query.sql [hunk 2]
```diff
@@ -47,3 +45,4 @@ GROUP BY
     machine_type
 ORDER BY
     count DESC
```
Comment (malfet) [inline torchci/clickhouse_queries/queued_jobs_by_label/query.sql:47]: Don't you need a semicolon here? Or is `SETTINGS` a keyword to `SELECT`?)
```suggestion
    count DESC;
```

File: torchci/clickhouse_queries/queued_jobs_by_label/query.sql [hunk 3]
```diff
@@ -47,3 +45,4 @@ GROUP BY
...
+SETTINGS final = 1
```
Comment (huydhn) [inline torchci/clickhouse_queries/queued_jobs_by_label/query.sql:48]: TIL!

**Review by huydhn — APPROVED** [review]

**Review by clee2000 — APPROVED** [review]

**Review by malfet — APPROVED** [review]

File: torchci/clickhouse_queries/queued_jobs_by_label/query.sql (hunk 1 above)
Comment (kit1980) [reply torchci/clickhouse_queries/queued_jobs_by_label/query.sql:2]: It's valid, but needs to enable a setting for multiple statements we probably don't want to enable.
It works in CH UI.

File: torchci/clickhouse_queries/queued_jobs_by_label/query.sql (hunk 2 above)
Comment (kit1980) [reply torchci/clickhouse_queries/queued_jobs_by_label/query.sql:47]: `SETTINGS` is a part of `SELECT`
https://clickhouse.com/docs/en/sql-reference/statements/select/from#final-modifier

**Pull request merged**